FreeRTOS/FreeRTOS-Kernel
# Remove stdint.h in stream buffer file

Description
-----------
This PR removes the superfluous inclusion of <stdint.h> in streambuffer.c file , as stdint gets included in FreeRTOS.h

Test Steps
-----------
Build and Run [Demo](https://github.com/FreeRTOS/FreeRTOS/tree/main/FreeRTOS/Demo/CORTEX_MPU_M7_NUCLEO_H743ZI2_GCC_IAR_Keil/Projects/GCC)  on STM32 platform.

Checklist:
----------





By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
